stories/showcase.stories.js

```diff
@@ -0,0 +1,62 @@
+import React, { useEffect, useState } from "react";
+import { storiesOf } from "@storybook/react";
+
+import { Showcase } from "@commons-ui/core";
+
+
+function timeStamp(timestamp) {
+  return new Date(timestamp).toLocaleString("en-GB", {
+    year: "numeric",
+    day: "2-digit",
+    month: "short",
+  });
+}
+
+function useStories(url) {
+  const urlJson = `https://corsio.devops.codeforafrica.org/?${url}?format=json`;
+  const [reports, setReports] = useState([]);
+  useEffect(() => {
+    async function fetchData() {
+      const response = await fetch(urlJson);
+      const jsonClean = await (await response.text()).replace(
+        '])}while(1);</x>',
+        ''
+      );
+      const json = await JSON.parse(jsonClean);
+      const reportStreamItems = await json.payload.streamItems;
+      const reportItems = [];
+      await reportStreamItems.forEach(function extractReports(
+        reportStreamItem
+      ) {
+        reportItems.push(
+          json.payload.references.Post[reportStreamItem.postPreview.postId]
+        );
+      });
+      setReports(reportItems);
+    }
+    fetchData();
+  }, [url, urlJson]);
+
+  return [reports];
+}
+
+function show() {
+  const [posts] = useStories('https://pesacheck.org/tagged/public-finance');
+  posts.forEach(function fixTimestamp(post) {
```

Comment:
`virtuals.previewImage` contains just the `path` without `hostname` causing images to fail on my end..